src/components/conversation-card/ConversationCard.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div class="conversation-card" @click="onClick">
+
+    <div class="avatar-container">
+      <avatar :image="image" />
+    </div>
+
+    <template v-if="!loading">
+      <div class="first">
+        <p class="txt-4 bold">
+          {{ userName }}
+        </p>
+        <p
+          v-if="isSmallOrDown"
```

Comment:
@KevinTss can we not solve this with CSS? We should be able to use the media query and change the CSS based on the media. Also see https://stackoverflow.com/questions/3194610/conditional-statement-for-screen-resolution

Reply:
We use both, depend of case we use css and for other JS
I always use media queries only to re-define some style, but to hide element I choose v-if in vue to avoid the component to build the DOM and increaes performance if it's on mobile

Reply:
I think the query approach is better, what if another platform would be added in the future? Then we have extra if statements and the readability of the code will be worse.

Reply:
Is 2 different things here, you have the JS way to make the plateform able to render things differently depend of the screen width. And you have the style who is responsible to give some style, this one can have some difference depend of the screen width too of course